### PR TITLE
code review for apple project - backgrounds and gradients

### PR DESCRIPTION
**Pull request by sheylaPozo**
This is a personal project, and I wanted to receive your feedback on it. Thank you.

### CODE REVIEW

**Review by mosesogwo — CHANGES_REQUESTED** [review]
### HI @sheylaPozo, nice work :+1:. Let us make the following changes to this project.

1. The page body should take 100% of the page width.

2, You have an html validation error. Check your markup at https://validator.w3.org/#validate_by_upload

3. Add a better description of the project in your readme. **solo project N°1** is not a good description.

4. These links in the readme are leading no where. Update them
image



5. Enclose main content of page in `<main>` tags. I see you have already used `<nav>` and `<footer>` tags.

6. Turn on stickler for this project. No checks were run.

**For future projects, take note of the following:**

1. Commit more regularly

2. In your pull request description, give a summary of the work done under that pull request that is to be reviewed.

_**Kindly make these changes and submit another code review request.**_

**Cheers:smile:**

### Status: Changes requested.

**Comment by sheylaPozo** [comment]
Hey, I made some changes, can you please review my code?

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Hi @sheylaPozo 
- Great job making some of the changes requested. However, for this project to be approved, you have to make all changes.
- [x] As mentioned by @mosesogwo, the links in your readme file are not working, fix this.
- [x] Please **do not** make any changes in Linter config files as mentioned on [Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png), use the files provided by microverse, you can find them [here](https://github.com/microverseinc/linters-config/tree/master/css).
- [x] As mentioned by @mosesogwo, give a summary of the work you have done in your pull request description.
### Kindly make these changes and request for another code review.
### Status: Changes required
_Feel free to reach out to me on slack(Fatima), if you have any questions, I'd be happy to help._

**Comment by sheylaPozo** [comment]
Hello, I made some changes in the work building with backgrounds and gradients, which is about learning the same, by cloning an apple old website and learning how to use backgrounds behind text and gradients using HTML and CSS.

**Comment by sheylaPozo** [comment]
Hello, I made some changes in the work building with backgrounds and gradients, which is about learning the same, by cloning an apple old website and learning how to use backgrounds behind text and gradients using HTML and CSS.

File: assets/style.css [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 534px;
+  background-image: url("hiking.jpg");
+  background-size: 1440px 1315px;
+  background-position-y: 534px;
+  width: 100%;
```
Comment (euqueme) [inline assets/style.css]: - [x] The selectors from line #58 to line #62 don't do any effect on your page so please remove them

File: README.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ solo project N°1
 
 ---
 
-![Example](https://web.archive.org/web/20140301004610/http://www.apple.com/)
+![Example](http://archive.li/UW4oR)
```
Comment (euqueme) [inline README.md]: - [x] The link its broken on line #12, please remove the `!` sign in the beginning of the line so it looks like this:

`[Example](http://archive.li/UW4oR)`

File: assets/style.css [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+nav ul li {
+  display: inline-block;
+  margin-left: 10px;
+  padding-right: 10px;
+  border-right: 1px ridge gray;
+}
+
+.font-x {
+  font-size: 0.7em;
+  color: black;
```
Comment (euqueme) [inline assets/style.css]: remove line #238 ( `color: black;`) as you are already using it in the class **.black-x**

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @sheylaPozo :lollipop: 

### Thank you for doing the changes requested by the previous reviewers :medal_sports: your project is looking better and better every time :eyes:  there are still a couple of changes left to make :hammer: 

## Footer

- [x] your footer links need to be in the center of the page (not centered) they are leaning to the left, also remove the underline from the links as they should only appear when the links are hovered like the original page

- Yours 

image

- Original

image

## Readme

- [x] You have a broken link in your Readme I have commented your code with a suggestion on how you can fix it :arrow_down_small: 

## CSS Good Practices

- [x] Some selectors within your CSS file don't make any effect in your HTML file please remove them as not only is good practice to have only the selectors we need also it'll make your page load faster. I've also have commented on your code where selectors should be removed :arrow_double_down: 

## Building with Backgrounds

- [ ] [OPTIONAL] This project is all about building with backgrounds and gradients so please change the product images to be `<div>` tags with background images.

### That's it! :raised_hands: 

### Please :pray: make the changes :memo:  and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by sheylaPozo** [comment]
I finished with the changes in backgrounds and Gradients project, I would like to get a review, thanks.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴 

Hi @sheylaPozo, great job fixing the recommended changes 👍 There are a few changes to get the project approved. Kindly fix comment below:
- [HTML markup](https://validator.w3.org/)  returned some errors 
sa

Kindly fix the markup error and submit code review 😃 
Happy coding 💻 
Daniel Larbi Addo

**Comment by sheylaPozo** [comment]
I dont think is an error though, I named it footiers on purpose lol a word
that doesn't exist. Mm and I will check in the Footer Tag

**Comment by sheylaPozo** [comment]
I fixed the last two errors for my project backgrounds and gradients, and now I'm requesting a code review.
 Thanks.

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello @sheylaPozo ,  You have done a great job with this project. :clap:

File: assets/style.css (hunk 3 above)
Comment (sheylaPozo) [reply assets/style.css]: Thank you for everything, your suggestions were very accurate and exactly what I needed.

**Pull request merged**